engine/previews/citizens_advice_components/icons_preview/icons.html.haml

```diff
@@ -0,0 +1,134 @@
+-# haml-lint:disable ViewLength
+.cads-grid-container
+  .cads-grid-row
+    .cads-grid-col-8
+      %table.cads-table
```

Comment:
We could probably generate the rows in this table if they all follow the same pattern but I don't mind the longhand.